fast64_internal/sm64/sm64_objects.py

```diff
@@ -1403,6 +1420,784 @@ def draw_act(self, obj, layout, value):
+        else:
+            return self.collision_object or bpy.context.active_object
+
+    @property
+    def gfx_object(self):
+        if not self.export_gfx:
+            return None
+        if self.export_all_selected:
+            return self.context_obj or bpy.context.active_object
+        else:
+            return self.graphics_object or bpy.context.active_object
+
+    @property
+    def bhv_object(self):
+        if not self.export_bhv:
+            return None
+        if self.export_all_selected:
+            return self.context_obj or bpy.context.active_object
+        else:
+            return self.col_object or self.gfx_object or bpy.context.active_object
+
+    @property
+    def group_num(self):
+        if self.group_name == "common0":
+            return 1
+        else:
+            return int(self.group_name.removeprefix("group")) + 1
+
+    @property
+    def obj_name_col(self):
+        if self.export_all_selected:
+            return self.col_object.name
+        if not self.object_name and not self.col_object:
+            return ""
+        else:
+            return self.object_name or self.col_object.name
+
+    @property
+    def obj_name_gfx(self):
+        if self.export_all_selected:
+            return self.gfx_object.name
+        if not self.object_name and not self.gfx_object:
+            return ""
+        else:
+            return self.object_name or self.gfx_object.name
+
+    @property
+    def obj_name_bhv(self):
+        if self.export_all_selected:
+            return self.bhv_object.name
+        if not self.object_name and not self.bhv_object:
+            return ""
+        else:
+            return self.object_name or self.bhv_object.name
+
+    @property
+    def bhv_name(self):
+        return "bhv" + "".join([word.title() for word in toAlnum(self.obj_name_bhv).split("_")])
+
+    @property
+    def geo_name(self):
+        return f"{toAlnum(self.obj_name_gfx)}_geo"
+
+    @property
+    def collision_name(self):
+        return f"{toAlnum(self.obj_name_col)}_collision"
+
+    @property
+    def model_id_define(self):
+        return f"MODEL_{toAlnum(self.obj_name_gfx)}".upper()
+
+    @property
+    def export_level_name(self):
+        if self.level_name == "custom":
+            return self.custom_export_name
+        return self.level_name
+
+    def draw_generic_options(self, layout):
+        split = layout.split(factor=0.333)
+        col = split.column()
+        if self.export_col:
+            box = col.box()
+            box.prop(self, "export_col")
+            box.prop(self, "include_children")
+            box.prop(self, "export_rooms")
+        else:
+            col.prop(self, "export_col")
+        split.prop(self, "export_gfx")
+        if not self.export_all_selected:
+            split.prop(self, "export_bhv")
+        row = layout.row()
+        row.prop(self, "export_all_selected")
+        row.prop(self, "export_script_loads")
```

Comment:
Shouldn´t this toggle be put in the graphics column like you do for collision toggles